Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -2077,6 +2042,7 @@ public class BrowserViewController: UIViewController {
     }
     
     if let findInPageBar = findInPageBar {
+      updateFindInPageVisibility(visible: false)
```

Comment:
Added for the case when Zoom is activated while Find =-In page is activated and searching a text.